javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/PropagatorsInitializer.java

```diff
@@ -39,9 +42,25 @@
           .put(JAEGER, JaegerPropagator.getInstance())
           .put(OT_TRACER, OtTracerPropagator.getInstance())
           .put(XRAY, AwsXRayPropagator.getInstance())
+          .put(BAGGAGE, W3CBaggagePropagator.getInstance())
           .build();
 
-  /** Initialize OpenTelemetry global Propagators with propagator list, if any. */
+  /**
+   * Initialize OpenTelemetry global Propagators with propagator list, if any.
+   *
+   * <p>Because TraceMultiPropagator returns first successful extracted Context and stops further
+   * extraction next rules applied:
+   *
+   * <ul>
+   *   <li>W3CBaggagePropagator is added to DefaultContextPropagators to allow another propagator
+   *       extract Context.
+   *   <li>JaegerPropagator is added to DefaultContextPropagators because it extracts both Context
```

Comment:
Is it always added? Or only if `jaeger` value is provided?

Reply:
added `can be` ... `only`